app/src/main/java/com/github/se/travelpouch/ui/travel/EditTravelSettings.kt

```diff
@@ -111,68 +110,45 @@ fun EditTravelSettingsScreen(
-                                  Icons.Default
-                                      .PersonAddAlt1, // Replace with your mail icon resource
-                              contentDescription = "Mail Icon",
-                              modifier = Modifier.size(24.dp) // Adjust size as needed
-                              )
-                          Spacer(modifier = Modifier.width(8.dp)) // Space between icon and text
-                          Text(
-                              text = "Add User",
-                              style = MaterialTheme.typography.bodyLarge // Or customize further
-                              )
-                        }
+                  modifier = Modifier.testTag("manageParticipantsButton"))
 
-                    // Text("Add User")
-                  }
-              FloatingActionButton(
+              Spacer(modifier = Modifier.height(8.dp))
+
+              ExtendedFloatingActionButton(
+                  text = { Text("Import Email", modifier = Modifier.testTag("importEmailText")) },
+                  icon = { Icon(Icons.Default.MailOutline, contentDescription = "Import Email") },
                   onClick = {
                     clipboardManager.setText(
                         AnnotatedString("travelpouchswent+${selectedTravel!!.fsUid}@gmail.com"))
                     Log.d("EditTravelSettingsScreen", "Email copied to clipboard")
+                    toggled = !toggled
                   },
-                  containerColor = MaterialTheme.colorScheme.tertiaryContainer,
-                  contentColor = MaterialTheme.colorScheme.onTertiaryContainer,
-                  modifier = Modifier.testTag("importEmailFab")) {
-                    Row(
-                        verticalAlignment = Alignment.CenterVertically,
-                        modifier =
-                            Modifier.padding(
-                                horizontal = 12.dp) // Adjust padding for better alignment
-                        ) {
-                          Icon(
-                              imageVector =
-                                  Icons.Default.MailOutline, // Replace with your mail icon resource
-                              contentDescription = "Mail Icon",
-                              modifier = Modifier.size(24.dp) // Adjust size as needed
-                              )
-                          Spacer(modifier = Modifier.width(8.dp)) // Space between icon and text
-                          Text(
-                              text = "Import Email",
-                              style = MaterialTheme.typography.bodyLarge // Or customize further
-                              )
-                        }
-                  }
+                  modifier = Modifier.testTag("importEmailFab"))
             }
+          } else {
+            FloatingActionButton(
+                onClick = { toggled = !toggled }, modifier = Modifier.testTag("plusButton")) {
```

Comment:
Here, we can't press the plus again to dismiss. In terms of UX it feels wrong to me that one is stuck and has to choose an option to go back